library/src/main/java/com/schibsted/spain/barista/interaction/BaristaAutoCompleteTextViewInteractions.kt

```diff
@@ -1,14 +1,22 @@
 package com.schibsted.spain.barista.interaction
 
 import android.support.annotation.IdRes
-import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.widget.EditText
 import com.schibsted.spain.barista.internal.performAction
 import com.schibsted.spain.barista.internal.viewaction.AutoCompleteViewActions.replaceAutoComplete
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.anyOf
 
 object BaristaAutoCompleteTextViewInteractions {
 
     @JvmStatic
     fun writeToAutoComplete(@IdRes autoCompleteId: Int, text: String) {
-        withId(autoCompleteId).performAction(replaceAutoComplete(text))
+        val withId = withId(autoCompleteId)
+        val assignableFrom = isAssignableFrom(EditText::class.java)
```

Comment:
Personal opinion: I think that extracting these two matchers to a variable with the same name of the matcher itself only adds noise.
Just writing `val simpleMatcher = allOf(withId(autoCompleteId), isAssignableFrom(EditText::class.java))` seems easier to understand.

Reply:
i will agree if those are used just on `simpleMatcher`, but they are used also on `wrappedMatcher`